src/geo/map/plain-layer.js

```diff
@@ -11,6 +14,28 @@ var PlainLayer = LayerModelBase.extend({
     className: 'plain',
     color: '#FFFFFF',
     image: ''
+  },
+
+  initialize: function (attrs, options) {
+    attrs = attrs || {};
+    options = options || {};
+    if (!options.vis) throw new Error('vis is required');
+
+    this._vis = options.vis;
+
+    this.bind('change', this._onAttributeChanged, this);
```

Comment:
That's interesting. But we still need to explicitly do `view.stopListening(model);` (http://backbonejs.org/#Events-stopListening), don't we?